### ISSUE TITLE
Can elastalert query two different cluster with different ES versions?

### ISSUE THREAD

**Issue by nickbabkin** [Yelp/elastalert#2353]
Good evening!
I have two elastic clusters running ES versions 6.6 and 7.2. I'd like to query both in different rules, for example:
Rule1 - query elastic_6.6
Rule2 - query elastic_7.2

Is there any option for setting ES version in a rule config file or workaround that will allow me to achieve this without running two separate elastalert daemons?

Thank you!

**Comment by Qmando** [comment]
Yes. This works automatically. Specify a different es_host in a rule and it will check it's version.

**Comment by nickbabkin** [comment]
@Qmando awesome, thanks. Will try it out.

**Comment by Qmando** [comment]
Whichever is in config.yaml
